.github/workflows/publish.yml

```diff
@@ -128,4 +128,12 @@ jobs:
       - uses: ./.github/actions/update-workspaces-version
         with:
           version: ${{ steps.bump-dev-semver.outputs.new_version }}
-          ref: ${{ env.GITHUB_REF_NAME }}
+          ref: ${{ github.ref_name }}
+          push: false
```

Comment:
Not sure I understand this. why push=flase for update-workspaces-version, but push=true for update-devcontainer-jsons?

Reply:
just because the push should be done only once- and update-devcontainer-jsons is the last place we commit